src/index.js

```diff
@@ -1,11 +1,19 @@
 import './style.css';
 import logoImg from './img/logo.png';
 import displayItems from './modules/display.js';
+import { popup, fetchCocktailById } from './modules/details.js';
 
 const logo = document.querySelector('#logoImg');
-
 logo.src = logoImg;
 
+const itemSection = document.querySelector('#listItems');
+
+itemSection.addEventListener('click', async (e) => {
+  const { id } = e.target.parentElement.children[1];
```

Comment:
This is not working as it should be, when you click on the reservations, you are also open the popup window.
kindly try to change this